worldedit-core/src/main/java/com/sk89q/worldedit/command/ClipboardCommands.java

```diff
@@ -298,7 +270,25 @@ public void cut(
         copy.setSourceFunction(new BlockReplace(editSession, leavePattern));
         copy.setCopyingEntities(copyEntities);
         copy.setRemovingEntities(true);
+        createCopy(session, editSession, copyBiomes, mask, clipboard, copy);
+
+        if (!actor.hasPermission("fawe.tips")) {
+            actor.print(Caption.of("fawe.tips.tip.lazycut"));
+        }
+        copy.getStatusMessages().forEach(actor::print);
+        //FAWE end
+    }
+
+    private void createCopy(
+            final LocalSession session,
+            final EditSession editSession,
+            @Switch(name = 'b', desc = "Also copy biomes") final boolean copyBiomes,
+            @ArgFlag(name = 'm', desc = "Set the include mask, non-matching blocks become air", def = "") final Mask mask,
```

Comment:
The flags do not have any functionality here, are they intentional? Might be fine for documentation purposes, but if e.g. the description is duplicated, it might get outdated.

Reply:
They are used in the code below..?

Reply:
I meant the annotations, sorry

Reply:
Eh, either way somewhat out of scope. The ArgFlag does allow for a default value and stuff though, unless I'm misunderstanding

Reply:
Yes, they are relevant for command methods, but this isn't a command method. That's why I'm wondering why the annotations are used here.

Reply:
OHH, yeah definitely no need then. Completely missed that